policyengine_us/parameters/gov/states/pa/tanf/income/earned_deduction/personal_expenses/full_time_employment.yaml

```
@@ -0,0 +1,11 @@
+description: Working hour requirement defining someone is full-time employed is on average at least 30 hours per week
```

Include the government and context, avoid include values
```suggestion
description: Pennsylvania defines full-time employment if they work at least this many hours per week for the purposes of TANF.
```